next.config.js

```diff
@@ -4,11 +4,21 @@ const withSourceMaps = require('@zeit/next-source-maps')
 
 const camelCaseLoader = path.join(__dirname, 'webpack/camelcase-loader.js')
 
-const envConfig = require('./config')
+const envConfig = require('./env.config')
+const csp = require('./csp.config')
 
 const nextConfig = {
   env: envConfig,
 
+  async headers() {
+    return [
+      {
+        source: '/:path*',
```

Comment:
This rule doesn't cover `/` page. Maybe you could add another rule? I mean we don't use that page but it would be nice to cover it properly.  :)

Reply:
I changed it to regular expression but it doesn't look good to me. I will probably report this as a bug